test/extensions/retry/host/omit_host_metadata/config_test.cc

```diff
@@ -4,7 +4,8 @@
 
 #include "extensions/retry/host/omit_host_metadata/omit_host_metadata.h"
 
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/upstream/host.h"
+#include "test/test_common/utility.h"
```

Comment:
the build target doesn't depend on utility. transitive? is it needed?

Reply:
it's needed as it used `TestUtility::parseYaml`. it's indirectly included from upstream/mocks.h before